includes/RestApi/SiteGenController.php

```diff
@@ -39,6 +39,15 @@ public function register_routes() {
 				'permission_callback' => array( Permissions::class, 'rest_is_authorized_admin' ),
 			)
 		);
+		\register_rest_route(
+			$this->namespace,
+			$this->rest_base . '/get-customize-data',
```

Comment:
Can we remove `get` from the endpoint? \WP_REST_Server::READABLE signifies that it is a `GET` request.

Reply:
Done